src/components/pages/profile/SignOutLink/SignOutLinkContainer.js

```diff
@@ -24,7 +21,4 @@ export const mapDispatchToProps = dispatch => ({
   },
 })
 
-export default connect(
-  mapStateToProps,
-  mapDispatchToProps
-)(SignOutLink)
+export default connect(mapStateToProps, mapDispatchToProps)(SignOutLink)
```

Comment:
```suggestion
export default connect(null, mapDispatchToProps)(SignOutLink)
```
Si tu enlèves le mapStateToProps au dessus :)